cmd/kubeadm/app/util/system/package_validator_test.go

```diff
@@ -224,8 +228,14 @@ func TestValidatePackageVersion(t *testing.T) {
 			if test.err != nil {
 				if err == nil {
 					t.Errorf("%s: v.validate() is expected to fail.", test.desc)
-				} else if test.err.Error() != err.Error() {
-					t.Errorf("%s: v.validate(): err = %q, want = %q", test.desc, err, test.err)
+				} else if len(test.err) != len(err) {
```

Comment:
I think, that this `if` statement and the `else` case below, can be collapsed into a single block via `reflect.DeepEqual`

Reply:
@rosti so, my five minute update turned into a interesting 2 hour lesson I learned tonight.

`reflect.DeepEqual` does not work for type of `"github.com/pkg/errors"`
`reflect.DeepEqual` Does in fact work for type of `"errors"` 
I'm going to try and figure out why maybe some other night.

I have attached two demos so you can see for yourself! I have updated to the best alternative I could think of right then. let me know if you know if a better way and I can update it.

https://play.golang.org/p/W3RFfNL8saG `"hithub.com/pkg/errors"`
https://play.golang.org/p/zBMv2BFC0K0 `"errors"`

Reply:
Yeah, I missed, that these may be different backing types (hence DeepEqual won't work). Sorry about that!